dracut/30ignition/ignition-generator

```diff
@@ -64,12 +64,19 @@ DefaultDependencies=false
 
 Requires=local-fs-pre.target
 Before=local-fs-pre.target
+Before=ignition-disks.service
+Before=ignition-files.service
```

Comment:
These are redundant with the `After`s in those services.

Reply:
right, but aren't we trying to be explicit? I'd rather name the dependency both ways so it's clear to people reading.

Reply:
Eh, IMO the redundancy makes it more difficult to tell which directives actually matter.  But I don't feel strongly about it.

Reply:
...IOW, explicit is in the eyes of the beholder?  :eyes:

Reply:
:eye:

Reply:
> IMO the redundancy makes it more difficult to tell which directives actually matter.

I would agree, but don't feel strongly either. :)
(Note that it gets pulled into the transaction by those services in the first place, so it does feel like the canonical place for ordering should be there).